### DISCUSSION TITLE
switching columns in detail view

### DISCUSSION THREAD

**Discussion by mgkai** [JPTGamesAndApps/ShresthaFiles#3] [Ideas]
I find it very useful to have pulled Date column right next to Name:

![image](https://user-images.githubusercontent.com/6969769/107440658-3c5aff00-6b34-11eb-8672-8e58b85e13f4.png)

Who else would find it worthwhile to have that added some time in the future?
(unless of course it is already there and I just missed it...)

**Comment by JPTGamesAndApps** [comment]
Thank you for the feedback! Just making sure, you would like to have Name, then Date, then Type, and then Size, right? Currently, the columns are fixed and cannot be customized by users. 

Just noted that the order you are requesting is the order in the Windows Explorer. We will keep this feature in consideration!

**Comment by JPTGamesAndApps** [comment]
Version 0.35 that should be available in the next few days should have the Date column before the Type column.

**Comment by mgkai** [comment]
Sorry, I wanted to reply yesterday but my connection dropped. My preference is name -> date -> Size -> Type, I f you can make it freely arrangable in the future, it will be perfect for all tastes / different users. Thank you

**Reply by JPTGamesAndApps** [reply]
We will keep the customization option under consideration! However, it will not be the priority at this point.
